Task_DEV_4/Task_DEV_4/EntryPoint.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Rectangle rectangle = new Rectangle();
+            double length = 4;
+            double width = 7;
+            rectangle.SettingFieldValues(length, width);
+            double rectangleArea = rectangle.CalculateTheArea();
+            Console.WriteLine($"The area of a rectangle is {rectangleArea}");
+            double rectanglePerimetr = rectangle.CalculateThePerimeter();
+            Console.WriteLine($"The perimetr of a rectangle is {rectanglePerimetr}");
+            bool isNotSquare = rectangle.CheckForSquare();
```

Comment:
Better not to use inverse logic. Direct logic looks clearer. Instead of isNotSquare, use isSquare.

Reply:
Done.